lighthouse-core/test/results/sample_v2.json

```diff
@@ -1825,53 +1825,43 @@
         "items": [
           {
             "url": "http://localhost:10200/zone.js",
-            "totalBytes": 71654,
-            "totalMs": 409.9376550703614
```

Comment:
These are technically breaking changes, but the Ms that was being returned was next to useless. We can return the Ms from lantern instead, but they weren't being surfaced so didn't seem worth it.